runtime/themes/acme.toml

```diff
@@ -22,6 +22,9 @@
 "diagnostic.hint" = {bg="white", modifiers=["bold"]}
 "ui.bufferline" = { fg = "indent", bg = "acme_bar_bg" }
 "ui.bufferline.active" = { fg = "black", bg = "acme_bg" }
+"diff.plus" = {fg = "green"}
+"diff.delta" = {fg = "red"}
+"diff.minus" = {fg = "acme_bar_bg"}
```

Comment:
Same here, you probably want minus as red (deletion) and delta as something else (modification, usually purple? `acme_bar_bg` could work)